src/job/ZosJobsProvider.ts

```diff
@@ -675,24 +681,32 @@ export class ZosJobsProvider extends ZoweTreeProvider implements IZoweTree<IZowe
      * Adds a single session to the jobs tree
      *
      */
-    private async addSingleSession(zosmfProfile: IProfileLoaded) {
-        if (zosmfProfile) {
+    private async addSingleSession(profileLoaded: IProfileLoaded) {
+        if (profileLoaded) {
             // If session is already added, do nothing
-            if (this.mSessionNodes.find((tempNode) => tempNode.label.trim() === zosmfProfile.name)) {
+            if (this.mSessionNodes.find((tempNode) => tempNode.label.trim() === profileLoaded.name)) {
                 return;
             }
-            // Uses loaded profile to create a zosmf session with Zowe
-            const session = ZosmfSession.createBasicZosmfSession(zosmfProfile.profile);
+
+            let session;
+            try {
+                // Uses loaded profile to create a zosmf session with Zowe
+                session = await getValidSession(profileLoaded, profileLoaded.name, false);
+            } catch (error) {
+                // When no password is entered, we should silence the error message for not providing it
+                // since password is optional in Zowe Explorer
+                if (error.message !== "Must have user & password OR base64 encoded credentials") { await errorHandling(error); }
```

Comment:
This should be checking for an error code. Another developer may change the text of an error message or add extra space/new line in the future breaking this logic not realizing another piece of code somewhere checks for this exact string. 
I suggest you add error codes to a special constants file.

Reply:
The error code for this message is undefined:
![image](https://user-images.githubusercontent.com/58596139/94525888-06784680-0235-11eb-97d8-a427192e7726.png)

Reply:
Seems like we should open an issue in the SDK/CLI repo?

Reply:
@katelynienaber You can set an error yourself in the code that throws that (I assume code that sets this string). If you decide to continue checking the string it should be made a constant, otherwise the logic is very fragile for the reasons mentioned above.

Reply:
Strings are also localized in the code base and you're checking for the English string.

Reply:
I opened an issue for missing error code here: [https://github.com/zowe/zowe-cli/issues/832](https://github.com/zowe/zowe-cli/issues/832)